modules/nf-core/emu/abundance/meta.yml

```diff
@@ -60,7 +60,7 @@ output:
         Groovy Map containing sample information
         e.g. [ id:'test', single_end:false ]
       pattern: "*{.sam}"
-  - results/*.sam:
+  - '*.sam':
       type: map
       description: |
         Groovy Map containing sample information
```

Comment:
Something has gone wrong here, copy paste error? Descirption should be of the file

Reply:
Updated this now.